boards/feather_m0/Cargo.toml

```diff
@@ -37,17 +37,23 @@ version = "0.3"
 optional = true
 
 [dev-dependencies]
-cortex-m-rtic = "1.0"
+embassy-executor = { version = "0.4.0", features = ["arch-cortex-m", "executor-thread", "task-arena-size-64"] }
```

Comment:
somehow this prevented the metro_m4 from compiling even though it's in an entirely separate bsp.  I changed it to 0.6.2 locally.

Reply:
Yeah looks like `0.4.0` was yanked. Fixed now